docs/user_guide/database/garbage_collection.md

```diff
@@ -204,10 +270,28 @@ merlin database gc --skip-runs --dry-run
 merlin database gc --skip-workers --skip-studies
 ```
 
+## Best Practices for Distributed Environments
+
+When working across multiple machines with different filesystem access:
+
+1. **Run from a machine with broad filesystem access**: For the most comprehensive cleanup, run garbage collection from a machine that has access to the most access shared file systems.
```

Comment:
Seems like 2 and 3 are probably the most important?  I.e. those two are what you should always do (regardless of how many machines you have to use) before falling back to manual deletion/cleanup.   This one seems more like: know your filesystem topologies/mounts, with a bonus being if you have on that can actually mount everything to do fewer iterations on 2 and 3?